include/platform/radio.h

```diff
@@ -70,7 +70,7 @@ enum
     kMaxPHYPacketSize        = 127,                       ///< aMaxPHYPacketSize (IEEE 802.15.4-2006)
     kPhyMinChannel           = 11,                        ///< 2.4 GHz IEEE 802.15.4-2006
     kPhyMaxChannel           = 26,                        ///< 2.4 GHz IEEE 802.15.4-2006
-    kPhySupportedChannelMask = 0xffff << kPhyMinChannel,  ///< 2.4 GHz IEEE 802.15.4-2006
+    kPhySupportedChannelMask = 0x07fff800,                ///< 2.4 GHz IEEE 802.15.4-2006
```

Comment:
Yes. Indeed! For some reason I thought it was going beyond 26!
I will close this PR.